lexicons/com/atproto/identity/sendPlcOp.json

```diff
@@ -0,0 +1,20 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.identity.sendPlcOp",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Validates a PLC operation to ensure that it doesn't violate a service's constraints or get the identity into a bad state, then submits it to the PLC registry",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["op"],
```

Comment:
I would probably expand all these "op" to "operation". eg the plc directory APIs have full "operation".

but not a strong feeling.

Reply:
eh yeah probably a good idea. generally we don't shoot for brevity in lexicon 😛